reports/api/ldap-authenticate.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env sh
+
+set -e
+
+if [ $# -ne 2 ]; then
+    echo "No username/password provided or too many parameters passed."
+    exit 1
+fi
+
+ldap_schema=${NETHVOICE_LDAP_SCHEMA:?No LDAP schema provided.}
+
+if [ "$ldap_schema" = "rfc2307" ]; then
+    exec ldapsearch \
+        -x \
+        -s base \
+        -b "$NETHVOICE_LDAP_BASE" \
+        -H "ldap://$NETHVOICE_LDAP_HOST:$NETHVOICE_LDAP_PORT" \
+        -D "uid=$1,ou=People,$NETHVOICE_LDAP_BASE" \
+        -w "$2" >  /dev/null
```

Comment:
I fear that this would break if the password contains some special chars like `$`, `"` or `&`.
Is it possible to read the pass from stdin?

Reply:
That's a thing that @Stell0 pulled up at first glance of the previous draft, I've run couple of tests and it seems that there's no issue with special characters, I'll run more test to be sure.

Reply:
After the tests, if everything is good, feel free to close the conversation.

Reply:
All good, parameter expansion escapes characters correctly